tests/relational_tests.rs

```diff
@@ -655,8 +655,8 @@ pub async fn linked() -> Result<(), DbErr> {
         name: String,
     }
 
-    let baker_query = Baker::find().find_also_linked(baker::BakedForCustomer);
-    let baked_for_customers: Vec<(BakerLite, Option<CustomerLite>)> = baker_query
+    let baked_for_customers: Vec<(BakerLite, Option<CustomerLite>)> = Baker::find()
```

Comment:
thanks, that was an oversight from my testcode, where I wanted to find out where the error actually comes from. But obviously it is not needed to stay that way.

Reply:
No worries! :)